README.md

```diff
@@ -29,3 +29,31 @@ Visit our **readthedocs** page for other setup and configuration at:
 <https://treeherder.readthedocs.io>
 
 File any bugs you may encounter [here](https://bugzilla.mozilla.org/enter_bug.cgi?product=Tree+Management&component=Treeherder).
+
+
+#### Contributing
+
+Firstly, ask if the bug is available and manifest your interest in contributing to it. One of the maintainers will **assign** you the bug. Only then you should work on it.
+
```

Comment:
"assign the bug to you".

Reply:
Also change in the first sentence." If a bug is not assigned to someone, you can request the bug be assigned to you. You should "need info" the component owner with your request. If you do not receive a response within 2-3 days, you can follow up in the #treeherder matrix channel." (feel free to clarify/improve that).